### PR TITLE
* itextsharp.csproj: switching to a more official microsoft version

### PR DESCRIPTION
**Pull request by orellabac**
Microsoft has release System.Drawing.Common which seems like a more "official" version for 
System.Drawing on .net core
I also made some minor changes to remove warnings on the following files:
* Image.cs:
* BaseFont.cs:
* PdfPTable.cs:
* RtfDestinationShppict.cs:

### CODE REVIEW

File: src/core/iTextSharp/text/rtf/parser/destinations/RtfDestinationShppict.cs [hunk 1]
```diff
@@ -74,7 +74,7 @@ public class RtfDestinationShppict : RtfDestination {
         private StringBuilder buffer = new StringBuilder();
 
         /* picttype */
-        private int pictureType = Image.ORIGINAL_NONE;
+      //  private int pictureType = Image.ORIGINAL_NONE;
```
Comment (bmeredith) [inline src/core/iTextSharp/text/rtf/parser/destinations/RtfDestinationShppict.cs]: In order to remain true to the original library, this should be remain uncommented.

File: src/core/iTextSharp/text/rtf/parser/destinations/RtfDestinationShppict.cs [hunk 2]
```diff
@@ -339,14 +338,14 @@ public class RtfDestinationShppict : RtfDestination {
...
                 if (ctrlWordData.ctrlWord.Equals("pict")) { result = true;}
-                if (ctrlWordData.ctrlWord.Equals("emfblip")) { result = true; pictureType = Image.ORIGINAL_NONE;}
-                if (ctrlWordData.ctrlWord.Equals("pngblip")) { result = true; pictureType = Image.ORIGINAL_PNG;}
-                if (ctrlWordData.ctrlWord.Equals("jepgblip")) { result = true; pictureType = Image.ORIGINAL_JPEG;}
-                if (ctrlWordData.ctrlWord.Equals("macpict")) { result = true; pictureType = Image.ORIGINAL_NONE;}
-                if (ctrlWordData.ctrlWord.Equals("pmmetafile")) { result = true; pictureType = Image.ORIGINAL_NONE;}
-                if (ctrlWordData.ctrlWord.Equals("wmetafile")) { result = true; pictureType = Image.ORIGINAL_WMF;}
-                if (ctrlWordData.ctrlWord.Equals("dibitmap")) { result = true; pictureType = Image.ORIGINAL_NONE;}
-                if (ctrlWordData.ctrlWord.Equals("wbitmap")) { result = true; pictureType = Image.ORIGINAL_BMP;}
+                if (ctrlWordData.ctrlWord.Equals("emfblip")) { result = true; /*pictureType = Image.ORIGINAL_NONE;*/}
```
Comment (bmeredith) [inline src/core/iTextSharp/text/rtf/parser/destinations/RtfDestinationShppict.cs]: These should also remain uncommented.

File: src/core/itextsharp.csproj [hunk 3]
```diff
@@ -18,9 +18,9 @@
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'net40' Or '$(TargetFramework)' == 'net45' ">
     <PackageReference Include="BouncyCastle" Version="1.8.1" />
+    <PackageReference Include="System.Drawing.Common" Version="4.5.0-preview1-26216-02" />
```
Comment (bmeredith) [inline src/core/itextsharp.csproj]: System.Drawing.Common should go under the netstandard2.0 ItemGroup as it is not compatible with net40 or net45.

**Review by bmeredith — CHANGES_REQUESTED** [review]
Thank you for the PR, I greatly appreciated it! 

Once the code I've made comments on are updated, I'll get your changes merged in and update the nuget package.

**Comment by orellabac** [comment]
I have updated the branch. 
Is it ok If I leave the comments but use a pragma warning disable to get rid of one of compiler message?

**Review by bmeredith — APPROVED** [review]

**Comment by bmeredith** [comment]
I've merged these changes and updated the nuget package to version 1.1.

Thanks again!

**Pull request merged**